src/Compilers/Core/Portable/Syntax/SyntaxNode.Iterators.cs

```diff
@@ -247,15 +243,15 @@ public void PushChildren(SyntaxNode node, Func<SyntaxNode, bool> descendIntoChil
                 }
             }
 
-            public void PushLeadingTrivia(ref SyntaxToken token)
+            public void PushLeadingTrivia(in SyntaxToken token)
             {
-                _triviaStack.PushLeadingTrivia(ref token);
+                _triviaStack.PushLeadingTrivia(in token);
```

Comment:
❓ Why explicit? At this point it behaves like passing by value which seems to be less noisy.

Reply:
It was explicit `ref` originally. I assumed that `ref` was used for performance which makes it important that we continue passing the token by direct reference. That is where you use `in` at the call site - to make sure there is no copying.

Basically - I used the "ref for performance reasons" pattern as an indication of perf-sensitive codepaths and used explicit `in` there. 
I think it makes sense to prevent unexpected perf regressions or at least as a self-documenting way to say that avoiding copies is very important.